src/Admin/Input/Input.php

```diff
@@ -51,6 +51,16 @@ class Input extends Form implements InputInterface {
 	 */
 	protected $value;
 
+	/**
+	 * @var bool
+	 */
+	protected $is_readonly = false;
```

Comment:
The reason I didn't do it is because the rest of the files (and other files like this one) have not the type set neither. So I bet for consistency in regards this topic.